content/en/04-tutorials/01-pixie-101/07-kafka-monitoring.md

```diff
@@ -113,9 +113,9 @@ This script is useful for inspecting a specific request or filtering all request
 
 ## Producer Consumer Latency
 
-Producer consumer latency is the time between when a producer pushes a message to a topic to when it is fetched by a consumer.
+Producer consumer latency is the time between when a producer pushes a message to a topic to when it is fetched by a consumer. This latency is important to monitor because many incidents begin with consumer lag.
 
-It's important to monitor this latency because many incidents begin with consumer lag.
+A large consumer lag in one component can mean that it is out of sync with other components. In our demo application, the `invoicing` service is not consuming messages from the `order` topic as quickly as the `shipment` service is. If you check the application front-end, you will see orders on the Shipment page that are not present on the Invoicing page. If not addressed, this application may fail to create invoices for all of the orders it receives and lose out on money owed for the orders that  successfully shipped.
```

Comment:
`shipment` service -> `shipping` service
There is an extra space between `that` and `successfully` on the last sentence.